### ISSUE TITLE
no internert

### ISSUE THREAD

**Issue by akbar9000** [EvilFreelancer/docker-routeros#2]
hi..
I installed and logged in ROS but no internet in router..where is problem

**Comment by EvilFreelancer** [comment]
Hello @akbar9000 !

You probably saw in readme of this project what this project is was created for test purpose only, in particular for testing [API library](https://github.com/EvilFreelancer/routeros-api-php) and not optimized for real work.

But, one of GitHub members with nick name @reinethernal created issue about additional ports from host machine to container interfaces, @reinethernal used `network_mode: "host"` option of docker-compose and created bridges on the host machine, then connected the necessary interfaces from container and host with these bridges, details here #1.

@akbar9000 you probably can make something simmilar, in brodge mode with host interfaces you don't need to worry about routing, eveyrthing will work as normal nerwork device. 

I hope my advice will help you :)

**Comment by akbar9000** [comment]
OK thanks..if you have got internet inside ROUTEROS please tell how..I will try @reinethernal method

**Comment by victorzzt** [comment]
Hi, first thanks for the image, all winbox and all internal functions works very well.
I got here because of the same problem.

I tried to solve it, but failed miserably, so want to ask you guys advice am I on the right track.

So right now I'm trying a not scalable way:
- First, I run the docker container with a custom created bridge (with static address)
- Then I try to get in and set the interface eth0 address to the container's static IP
- Use docker to copy the whole folder out and delete the container
- Add to the entrypoint.sh ifconfig container to an unusable ip address and switch the qemu to use tap network, map the routeros folder as volume when running.

```#!/bin/sh

# basically make it unusable
ifconfig eth0 173.11.0.1 netmask 255.255.255.255

sleep 5

qemu-system-x86_64 \
    -vnc 0.0.0.0:0 \
    -m 256 \
    -hda $ROUTEROS_IMAGE \
    -device e1000,netdev=net0 \
    -netdev tap,id=net0
```

And then the container keeps restarting. basically what I'm thinking is that the container host doesn't really need the ip address, and ros can takerover the container's interface and use the ip with full access.

Right now getting
```
ifconfig: SIOCSIFADDR: Operation not permitted
qemu-system-x86_64: could not open /dev/net/tun: No such file or directory
```
And I always thought in the container has root access.
I'm still new on this. Thanks in advance.

**Comment by EvilFreelancer** [comment]
Hello here! Small updated, now possible to map any part of routeros to host machine, plus issue with "no internet" from routeros is also fixed, please use at least `evilfreelancer/docker-routeros:6.46.5`
